packages/macros/src/glimmer/macro-maybe-attrs.ts

```diff
@@ -20,11 +20,11 @@ export function maybeAttrs(elementNode: any, node: any, builders: any) {
 
   if (result.value) {
     for (let bareAttr of bareAttrs) {
-      elementNode.attributes.push(builders.attr(bareAttr.original, builders.text('')));
+      elementNode.attributes.push(builders.attr(bareAttr.original, builders.text(''), bareAttr.loc));
```

Comment:
Some `macroMaybeAttrs` tests were failing due to some changes in the ember-template-compiler. 

[This PR](https://github.com/glimmerjs/glimmer-vm/pull/1266) added sorting logic to the printer which [sorts attributes](https://github.com/charlespierce/glimmer-vm/blob/b963b0955214166109a6560ea2c685a6a2b8ff22/packages/%40glimmer/syntax/lib/generation/printer.ts#L222) based on their `loc` data. Since the `macroMaybeAttrs` transform didn't attach `loc` information to the newly created attributes they were being moved to the front of the element due to this sorting. In particular, [this condition](https://github.com/charlespierce/glimmer-vm/blob/b963b0955214166109a6560ea2c685a6a2b8ff22/packages/%40glimmer/syntax/lib/generation/util.ts#L71) is hit because the attributes without `loc` data seem to default to a `0` value for `column`.

I reused the `loc` from the original source so that the printer maintains the correct order and the tests pass again.

Let me know if that wasn't the correct fix. :smile: